### ISSUE TITLE
No man pages...?

### ISSUE THREAD

**Issue by mariobl** [jimevins/glabels-qt#26]
Glabels 3.x has man pages for both the binary itself and the glabels-batch script. Admittedly, I haven't installed Glabels 4.x from source yet, but I can't found a man page source anywhere in the Git repo. OK, if we use it only via the AppImage, probably it doesn't make sense to call "man glabels-qt".

But especially for inclusion in Debian, man pages will be needed again. And once we will have a user manual, the man pages could be integrated into the translation stack. Maybe the man pages should be built from an XML source (not from Groff, this would need extra tools like po4a). Just an idea...

**Comment by jimevins** [comment]
I forgot you had done this.  I just checked in a set of man pages with commit c2d024f47ec8cac823f38a65091f1be34d6b5d53.  These are written in reStructuredText and converted using the sphinx toolchain.  See my comment in Issue #14.

**Comment by mariobl** [comment]
OK, looks fine so far, much better than my first attempt using Markdown. Bur not Markdown itself was the problem, but rather the peculiarities of po4a. I hope things will go better with the built-in i18n mechanism in Sphinx.

**Comment by mariobl** [comment]
I've tried to implement i18n support to the man pages. According to [1] I added "locale_dirs = ['locale/']" to user-docs/conf.py, but this doesn't seem to be sufficient. Cmake is unable to produce the "gettext" target in the Makefile. Would be nice to have the ability to create/update pot files soon...

BTW, as long as the Python module "sphinx-intl" is needed for creating the localized target documentation, all the doc files need to be created upstream. On my Archlinux system, that module [2] is available from the Arch Linux User Repository (AUR) only, but the "packages" there (which are in fact no more than spec files for building packages locally) cannot be considered as build dependencies for other packages which are to be build for the official Archlinux repositories. So Glabels-qt, once it is mature and complete enough for a stable release, wouldn't find the way into the official package repos, because it needs stuff from the AUR.

[1] http://www.sphinx-doc.org/en/master/usage/advanced/intl.html
[2] https://aur.archlinux.org/packages/python-sphinx-intl/
